prometheus_es_exporter/__init__.py

```diff
@@ -509,6 +519,14 @@ def cli(**options):
                                    '--indices-stats-mode must be "indices" for '
                                    '--indices-stats-indices to be used.')
 
+    executor = None
+    if options['threads']:
+        num_threads = int(options['threads'])
+        if num_threads < 1:
+            raise click.BadOptionUsage('threads',
+                                       '--threads must specify a number of threads greater than 0.')
+        executor = concurrent.futures.ThreadPoolExecutor(max_workers=num_threads)
```

Comment:
Since click is validating (and converting) the number of threads for us, we just need to handle `num_threads == 1`

```suggestion
    num_threads = options['threads']
    if num_threads > 1:
        executor = concurrent.futures.ThreadPoolExecutor(max_workers=num_threads)
```

Reply:
Awesome, thanks! (I've never used click before, as you can tell...)